spring-batch-docs/asciidoc/job.adoc

```diff
@@ -19,7 +19,7 @@ image::{batch-asciidoc}images/spring-batch-reference-model.png[Figure 2.1: Batch
 
 While the `Job` object may seem like a simple
 container for steps, there are many configuration options of which a
-developers must be aware . Furthermore, there are many considerations for
+developer must be aware. Furthermore, there are many considerations for
```

Comment:
I don't understand this point, the "a" is still there in the PR. The sentence is: "there are many configuration options of which a developer must be aware". Is this correct?

Reply:
It can be either "of which a developer must be aware" or "of which developers must be aware". The plural dictates whether "a" should be present. You could ditch the whole problem by replacing "there are many configuration options of which a developers must be aware" with "developers should be aware of many configuration options".

Reply:
> It can be either "of which a developer must be aware" or "of which developers must be aware".

ok, in this case, I kept the current "of which a developer must be aware"